back/src/main/resources/properties/notification-worker.yml

```
@@ -0,0 +1,42 @@
+    guardMs: 500
+    backfillMinutes: 10
+    ttl:
+      ontimeSec: 60
+      backfillSec: 600
+    lease:
+      ontimeSec: 60
+      backfillSec: 30
+  writer:
+    flush:
+      batch: 500
+      intervalMs: 200
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)